crt_portal/cts_forms/templates/privacy.html

```diff
@@ -7,6 +7,8 @@
 <title>{% trans "Civil Rights Division - Department of Justice - Privacy Policy" %}</title>
 {% endblock %}
 
+{% block menu %}{% endblock %}
```

Comment:
It feels like we should leave these here and use them as links back to the information on the landing page. Otherwise the privacy policy page is a bit of a dead-end

Reply:
Good idea; updated.